onelogin_rules/onelogin_user_account_locked.yml

```diff
@@ -0,0 +1,49 @@
+AnalysisType: rule
+Filename: onelogin_user_account_locked.py
+RuleID: OneLogin.UserAccountLocked
+DisplayName: OneLogin User Locked Rule Triggered
+Enabled: true
+LogTypes:
+  - OneLogin.Events
+Tags:
+  - OneLogin
+Severity: Low
+Description: >
+  User locked out of their account
+Reference: https://developers.onelogin.com/api-docs/1/events/event-resource
+Runbook: >
+  Investigate whether this was caused by expected action.
+Tests:
+  -
+    Name: User account locked via api.
+    ExpectedResult: true
+    Log:
+      {
+        'event_type_id': 531,
+        'actor_user_id': 123456,
+        'actor_user_name': 'Bob Cat',
+        'user_id': 123456,
+        'user_name': 'Bob Cat'
+      }
+  - 
+    Name: User account locked via api.
```

Comment:
While we don't enforce this in the CLI tool, I believe the backend behaves strangely when two unit tests have the same name. Would you mind:

1. Changing the names
2. Verifying if the backend does anything weird in the UI if you have two unit tests with the same name
3. If it does, put in an issue on the panther_analysis_tool to verify that test names are different

Thanks!

Reply:
Ok, I changed the names - also so the test output shows more clearly they are doing different things.  

The UI doesn't seem to have any issues with multiple unit tests with the same name.  I manually edited a rule's tests and it seemed to work (it saved/I was able to run the tests).  I also tried bulk uploading some rules with duplicate test names, and that also seems to have worked.